### PR TITLE
add EncryptInsertValuesTokenGeneratorTest

### LINKED ISSUE #15821: Add unit test for EncryptInsertValuesTokenGenerator
Hi community,

We need to add some unit test for EncryptInsertValuesTokenGenerator.

- EncryptInsertValuesTokenGenerator#isGenerateSQLToken
- EncryptInsertValuesTokenGenerator#generateSQLTokens

### PR DESCRIPTION
**Pull request by sunkai-cai**
Fixes #15821 , #15819

Changes proposed in this pull request:
-  add EncryptInsertValuesTokenGeneratorTest
-  add EncryptForUseDefaultInsertColumnsTokenGeneratorTest.assertGenerateSQLTokenFromPreviousSQLTokens
-  fix NPE

### CODE REVIEW

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptInsertValuesTokenGenerator.java [hunk 1]
```diff
@@ -77,6 +77,9 @@ public InsertValuesToken generateSQLToken(final InsertStatementContext insertSta
     }
     
     private Optional<SQLToken> findPreviousSQLToken(final Class<?> sqlToken) {
+        if (null == previousSQLTokens) {
```
Comment (TeslaCN) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptInsertValuesTokenGenerator.java:80]: Why change the production code?

Comment (sunkai-cai) [reply shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptInsertValuesTokenGenerator.java:80]: a NPE when previousSQLTokens is null

Comment (sunkai-cai) [reply shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptInsertValuesTokenGenerator.java:80]: EncryptInsertValuesTokenGenerator need to setPreviousSQLTokens , if not previousSQLTokens will be null.

Comment (sunkai-cai) [reply shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptInsertValuesTokenGenerator.java:80]: I'm not sure previousSQLTokens  must be set in another place

Comment (TeslaCN) [reply shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptInsertValuesTokenGenerator.java:80]: > a NPE when previousSQLTokens is null

previousSQLTokens is Collection, null value is not allowed for a Collection in ShardingSphere's code style.

Comment (sunkai-cai) [reply shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptInsertValuesTokenGenerator.java:80]: ok, I remove it.

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptGeneratorBaseTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        List<ExpressionSegment> result = new ArrayList<>(4);
+        result.add(new ParameterMarkerExpressionSegment(0, 0, 1));
+        result.add(new ParameterMarkerExpressionSegment(0, 0, 2));
+        result.add(new ParameterMarkerExpressionSegment(0, 0, 3));
+        result.add(new ParameterMarkerExpressionSegment(0, 0, 4));
+        return result;
+    }
+    
+    protected static List<SQLToken> getPreviousSQLTokens() {
+        List<SQLToken> result = new ArrayList<>(2);
```
Comment (TeslaCN) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptGeneratorBaseTest.java:89]: Why the size is 2 here?

**Review by TeslaCN — APPROVED** [review]

**Pull request merged**